lib/mayaHydra/hydraExtensions/adapters/materialNetworkConverter.cpp

```diff
@@ -395,6 +395,38 @@ class MayaHydraUvAttrConverter : public MayaHydraMaterialAttrConverter
     const VtValue _value;
 }; // namespace
 
+
+class MayaHydraOpenPBREmissionColorMaterialAttrConverter : public MayaHydraComputedMaterialAttrConverter
+{
+public:
+    SdfValueTypeName GetType() override { return SdfValueTypeNames->Vector3f; }
+
+    VtValue GetValue(
+        MFnDependencyNode&      node,
+        const TfToken&          paramName,
+        const SdfValueTypeName& type,
+        const VtValue*          fallback = nullptr,
+        MPlugArray*             outPlug = nullptr) override
+    {
+        GfVec3f emissionColorVec3f(1.0f, 1.0f, 1.0f);
+        VtValue emissionColor = MayaHydraMaterialNetworkConverter::ConvertMayaAttrToValue(
+            node, "emissionColor", SdfValueTypeNames->Vector3f, fallback, outPlug);
+        if (emissionColor.IsHolding<GfVec3f>()) {
+            emissionColorVec3f = emissionColor.UncheckedGet<GfVec3f>();
+        }
+
+        // TODO: Use emissionWeight directly when it's available in Maya
+        float emissionWeightFloat = 0.0f;
+        VtValue emissionLuminance = MayaHydraMaterialNetworkConverter::ConvertMayaAttrToValue(
+            node, "emissionLuminance", SdfValueTypeNames->Float, fallback, outPlug);
+        if (emissionLuminance.IsHolding<float>()) {
+            emissionWeightFloat = emissionLuminance.UncheckedGet<float>() / 1000.f;
```

Comment:
Could you please add a comment as why you are dividing by 1000 ?

Reply:
This is a temp simulation currently used. also applied in MaterialX's translation, will add a comment